db/021-add-siren-particle-id.sql

```diff
@@ -0,0 +1,20 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 20;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE locations ADD COLUMN siren-particle-id TEXT;
```

Comment:
Oh good. I'm glad this worked for you this time!

Can you please use underscores instead of hyphens in the column name. It will be more consistent with our other column names and also sometimes hyphens are interpreted as minus signs and we might get some weird attempts at subtraction when using hyphens.